ios/TabViewImpl.swift

```diff
@@ -155,6 +157,17 @@ extension View {
         .ignoresSafeArea(.container, edges: .horizontal)
         .ignoresSafeArea(.container, edges: .bottom)
         .frame(idealWidth: frame.width, idealHeight: frame.height)
+      }
+    }
+    
+    @ViewBuilder
+    func tabBarTranslucent(_ translucent: Bool) -> some View {
+        self
+            .onAppear {
+                UITabBar.appearance().isTranslucent = translucent
+            }
+            .onChange(of: translucent) { newValue in
+                UITabBar.appearance().isTranslucent = newValue
+            }
     }
```

Comment:
Looks like you are using 4 space indentation while the project is using 2 spaces, can you fix that? We probably need SwiftLint in the repo to catch this

Reply:
Added an empty setter in android and fixed indentation issues as well.

Thank you for suggestion @okwasniewski. Will keep these in mind in future PRs.